### PR TITLE
Update changelog for 0.3.0

### PR DESCRIPTION
**Pull request by ms-ati**
I'll rebase this after the "prepare for initial release" PR merges

### CODE REVIEW

File: CHANGELOG.md
Comment (jemmaissroff) [inline CHANGELOG.md:11]: Should we make this `2021-02-11`?

File: CHANGELOG.md
Comment (jemmaissroff) [inline CHANGELOG.md:52]: `[0.2.0]` --> `[0.3.0]`

**Review by jemmaissroff — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (ms-ati) [reply CHANGELOG.md]: Updated!

File: CHANGELOG.md
Comment (ms-ati) [reply CHANGELOG.md]: Oops good catch! Fixed!

**Review by jemmaissroff — APPROVED** [review]
LGTM once the first one merges!

**Pull request merged**
